### PR TITLE
Relax conda version requirements

### PR DESCRIPTION
**Pull request by jakirkham**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge/core @conda-forge/staged-recipes, any reason not to do this?

**Comment by beckermr** [comment]
There is a bug in conda/conda-build that was preventing builds from completing. @jjhelmus put in a PR to fix, but IDK the status.

**Comment by jakirkham** [comment]
Is there an issue upstream for tracking this? If not, would you be able to raise one?

**Comment by beckermr** [comment]
there is. I just lost it. :(

**Comment by beckermr** [comment]
https://github.com/conda/conda-build/issues/3735

**Comment by jakirkham** [comment]
No worries. Thanks!

Looks like PR ( https://github.com/conda/conda/pull/9289 ) landed for this recently, but has not been released yet.

**Comment by beckermr** [comment]
It appears the PR was merged (yay!) but it has not hit a conda release yet (:(). Once we do, we will still need to be at at least 4.7.13 since 4.7.11 and 4.7.12 break staged recipes.

**Comment by beckermr** [comment]
LOL. Same thoughts!

**Comment by jakirkham** [comment]
> Once we do, we will still need to be at at least 4.7.13 since 4.7.11 and 4.7.12 break staged recipes.

Having a lower bound as opposed to an upper bound would be a welcome change. 🙂

**Comment by jakirkham** [comment]
Do you know of any other issues still outstanding with this @beckermr?

**Comment by beckermr** [comment]
Nope. LGTM!

File: .appveyor.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ install:
     - cmd: appveyor-retry conda.exe update --yes --quiet conda
 
 
-    - cmd: appveyor-retry conda.exe install --yes --quiet "conda!=4.6.1,<4.7.11a0" conda-forge-pinning conda-forge-ci-setup=2.* networkx=2.3 "conda-build>=3.18"
+    - cmd: appveyor-retry conda.exe install --yes --quiet conda conda-forge-pinning conda-forge-ci-setup=2.* networkx=2.3 "conda-build>=3.18"
```
Comment (beckermr) [inline .appveyor.yml]:
```suggestion
    - cmd: appveyor-retry conda.exe install --yes --quiet "conda>4.7.12" conda-forge-pinning conda-forge-ci-setup=2.* networkx=2.3 "conda-build>=3.18"
```

File: .azure-pipelines/azure-pipelines-osx.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
   - script: |
       set -x -e
       source activate base
-      conda install -n base -c conda-forge --quiet --yes "conda!=4.6.1,<4.7.11a0" conda-forge-ci-setup=2 conda-build shyaml networkx=2.3 conda-forge-pinning
+      conda install -n base -c conda-forge --quiet --yes conda conda-forge-ci-setup=2 conda-build shyaml networkx=2.3 conda-forge-pinning
```
Comment (beckermr) [inline .azure-pipelines/azure-pipelines-osx.yml]:
```suggestion
      conda install -n base -c conda-forge --quiet --yes "conda>4.7.12" conda-forge-ci-setup=2 conda-build shyaml networkx=2.3 conda-forge-pinning
```

File: .azure-pipelines/azure-pipelines-win.yml [hunk 3]
```diff
@@ -44,7 +44,7 @@ jobs:
 
     - task: CondaEnvironment@1
       inputs:
-        packageSpecs: 'python=3.6 conda-build>=3.18 conda!=4.6.1,<4.7.11a0 conda-forge::conda-forge-ci-setup=2 networkx=2.3 conda-forge-pinning' # Optional
+        packageSpecs: 'python=3.6 conda-build>=3.18 conda conda-forge::conda-forge-ci-setup=2 networkx=2.3 conda-forge-pinning' # Optional
```
Comment (beckermr) [inline .azure-pipelines/azure-pipelines-win.yml]:
```suggestion
        packageSpecs: 'python=3.6 conda-build>=3.18 conda>4.7.12 conda-forge::conda-forge-ci-setup=2 networkx=2.3 conda-forge-pinning' # Optional
```

File: .circleci/build_steps.sh [hunk 4]
```diff
@@ -42,7 +42,7 @@ export CONDA_NPY='19'
 # Make sure build_artifacts is a valid channel
 conda index /home/conda/staged-recipes/build_artifacts
 
-conda install --yes --quiet "conda!=4.6.1,<4.7.11a0" conda-forge-ci-setup=2.* conda-forge-pinning networkx=2.3 "conda-build>=3.16"
+conda install --yes --quiet conda conda-forge-ci-setup=2.* conda-forge-pinning networkx=2.3 "conda-build>=3.16"
```
Comment (beckermr) [inline .circleci/build_steps.sh]:
```suggestion
conda install --yes --quiet "conda>4.7.12" conda-forge-ci-setup=2.* conda-forge-pinning networkx=2.3 "conda-build>=3.16"
```

File: .travis_scripts/build_all [hunk 5]
```diff
@@ -26,7 +26,7 @@ echo ""
 echo "Configuring conda."
 conda config --add channels conda-forge
 conda config --set show_channel_urls true
-conda install --yes --quiet "conda!=4.6.1,<4.7.11a0" conda-forge-ci-setup=2.* conda-forge-pinning networkx=2.3 "conda-build>=3.16"
+conda install --yes --quiet conda conda-forge-ci-setup=2.* conda-forge-pinning networkx=2.3 "conda-build>=3.16"
```
Comment (beckermr) [inline .travis_scripts/build_all]:
```suggestion
conda install --yes --quiet "conda>4.7.12" conda-forge-ci-setup=2.* conda-forge-pinning networkx=2.3 "conda-build>=3.16"
```

**Comment by jakirkham** [comment]
Thanks @beckermr! 😄

**Pull request merged**